packages/plugin-ext/src/plugin/languages/completion.ts

```diff
@@ -146,6 +146,10 @@ export class CompletionAdapter {
             };
         }
 
+        const tags = (!!item.tags || item.deprecated === true)
```

Comment:
I'm not sure about the condition. In the case of `item.tags === undefined` and `item.deprecated === true` there will be an exception:

```
> const item = {deprecated : true};
undefined
> const tags = (!!item.tags || item.deprecated === true) ? [...item.tags] : undefined;
Uncaught TypeError: item.tags is not iterable
```
does it mean that `item.tags` always has an array?
but if `item.tags` has some empty array but `deprecated` is true then how is it enough to later understand that this item is deprecated?

Reply:
I update the condition to be more robust (in the case of an empty array).

> does it mean that item.tags always has an array?

`item.tags` is `CompletionItemTag[] | undefined`.

Reply:
I still get an error when `item.tags` is `undefined`:

```
const item = {deprecated : true};
const tags = (!!item.tags?.length || item.deprecated === true) ? [...item.tags] : undefined;
VM35:2 Uncaught TypeError: item.tags is not iterable
    at <anonymous>:2:75
```

Reply:
Sorry, I must have messed up the variable when I initially tried to make the deprecated `deprecated` field supported as well.
It should be fine now, when either the `tags` or `deprecated` is truthy we will mark the entry as `deprecated` in the list.